mmv1/third_party/terraform/services/sql/resource_sql_user_test.go

```diff
@@ -55,6 +55,39 @@ func TestAccSqlUser_mysql(t *testing.T) {
 	})
 }
 
+func TestAccSqlUser_password_wo(t *testing.T) {
+	t.Parallel()
+
+	instance := fmt.Sprintf("tf-test-%d", acctest.RandInt(t))
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccSqlUserDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testGoogleSqlUser_password_wo(instance, "password"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckGoogleSqlUserExists(t, "google_sql_user.user1"),
+				),
+			},
+			{
+				// Update password
```

Comment:
here & other update checks, use the update plancheck: https://googlecloudplatform.github.io/magic-modules/test/test/#add-an-update-test

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/pull/13135/commits/eab0a5b48adcd42cd1ffd99b0c7dd85746b50610